scripts/cxg_admin.py

```diff
@@ -116,7 +116,10 @@ def transfer_collections(ctx, curr_owner, new_owner):
                 .filter(DbCollection.owner == curr_owner)
                 .update({DbCollection.owner: new_owner})
             )
+            session.commit()
             if updated > 0:
+                collections = session.query(DbCollection).filter(DbCollection.owner == new_owner).all()
+                click.echo(f"{new_owner} previously owned {len(new_owner_collections)}, they now own {len(collections)}. {updated} collections were updated")
```

Comment:
```suggestion
                click.echo(f"{new_owner} previously owned {new_owner_collections_count}, they now own {len(collections)}.)
```

Reply:
I would iterate over `updated` and print out the `collection.id` for all of the modified collections.